indico/htdocs/js/indico/modules/eventdisplay.js

```diff
@@ -81,26 +77,98 @@ $(document).ready(function() {
         } else {
             content[visible === undefined ? 'slideToggle' : visible ? 'slideDown' : 'slideUp']();
         }
-    }
+    };
 
-    $('a.js-show-note-toggle').on('click', function(e) {
-        e.preventDefault();
-        toggleNote($(this));
-    });
+    $(document).ready(function() {
+        $('.event-service-row > .trigger').on('click', function() {
+            var toggler = $(this);
+            toggler.siblings('.event-service-details').slideToggle({
+                start: function() {
+                    toggler.toggleClass('icon-expand icon-collapse');
+                },
+                duration: 'fast'
+            });
+        });
 
-    $('input.js-toggle-note-cb').on('change', function(e, immediate) {
-        toggleNote($(this), this.checked, immediate);
-    });
+        $('.event-service-row-toggle').on('click', function(e) {
+            e.preventDefault();
+            var toggler = $(this);
+            var toggler_button = $(this).parent().siblings('.trigger');
+            toggler.parent().siblings('.event-service-details').slideToggle({
+                start: function() {
+                    toggler_button.toggleClass('icon-expand icon-collapse');
+                },
+                duration: 'fast'
+            });
+        });
 
-    $('input.js-toggle-note-cb').trigger('change', [true]);
+        var gradientLayer = $('.participant-list-wrapper > .gradient-layer');
+        var heightControl = $('<div>', {'class': 'height-control'});
```

Comment:
apparently we don't really need the quotes around class anymore since all our supported browsers have ES5 engines